docs/dev/style-guide.rst

```diff
@@ -67,3 +71,98 @@ but even more importantly,
 it helps and includes readers by giving them quick and easy access to terms that they may be unfamiliar with.
 
 Use an external link or Intersphinx reference when a term is clearly defined elsewhere.
+
+Cross-references
+----------------
+
+We like to cross-reference other articles with a definition list inside a ``seealso::`` admonition box.
+It looks like this:
+
+.. code-block:: rst
+
+   .. seealso::
+
+      :doc:`/other/documentation/article`
+        You can learn more about <concept> in this (how-to/description/section/article)
+
+Diátaxis Framework
+------------------
+
+We apply the methodology and concepts of the Diátaxis Framework.
+This means that *both content and navigation path* for all sections should fit a single category of the 4 Diátaxis categories:
+
+* Tutorial
+* Explanation
+* How-to
+* Reference
+
+.. seealso::
+
+   `https://diataxis.fr/ <https://diataxis.fr/>`__
+     The official website of Diátaxis is the main resource.
+     It's best to check this out before guessing what the 4 categories mean.
+
+.. warning:: **Avoid minimal changes**
+
+   If your change has a high coherence with another proposed or planned change,
+   propose the changes in the same PR.
+
+   By multi-tasking on several articles about the same topic,
+   such as an explanation *and* a how-to,
+   you can easily design your content to end up in the right place *Diátaxis-wise*.
+   This is great for the author and the reviewers
+   and it saves coordination work.
+
+   Minimal or isolated changes generally raise more questions and concerns
+   than changes that seek to address a larger perspective.
+
+Explanation
+^^^^^^^^^^^
+
+* Introduce the scope: **“This article introduces ...”**
+  (write this as the very first thing,
+  then consider rewriting it or turning it into an internal comment afterwards).
+* Cross-reference the related How-to Guide.
+  Put a ``seealso::`` somewhere visible.
+  It should likely be placed right after the introduction,
+  and if the article is very short, maybe at the bottom.
+* Consider adding an Examples section
+* Can you add screenshots or diagrams?
+
+How-to Guides
+^^^^^^^^^^^^^
+
+* Title should begin with **“How to ...”**,
+  but navigation titles should not contain the “How to” part.
+* Introduce the scope: **“In this guide, we will…”**
+* Cross-reference related explanation.
```

Comment:
```suggestion
* `Cross-reference <#cross-references>`_ related explanation.
```

:D

Reply:
Right, haha, the other subsection came after this was written. Should this be 

```
:ref:`Cross-references`
```

?

Reply:
There is now a cross-reference to cross-references, so the point should be hopefully clear :)